builtin/mainmenu/dlg_config_world.lua

```diff
@@ -80,14 +83,18 @@ local function get_formspec(data)
 			end
 		end
 	end
-
+	if enabled_all then 
```

Comment:
Whitespace at end and you need to put one more indentation level on the following code.